src/test/java/spec/BaseSpec.java

```diff
@@ -1,26 +1,22 @@
 package spec;
 
-import io.github.bonigarcia.wdm.WebDriverManager;
+import helperClasses.SingletonBrowserClass;
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
 import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.chrome.ChromeDriver;
-
-import java.time.Duration;
 
 public class BaseSpec {
-    public WebDriver driver;
+    protected static WebDriver driver;
```

Comment:
Why is driver static?

Reply:
This shouldn't be static I agree. The on in Singleton class must be static. I fixed it